collective/js/throttledebounce/upgrades/20191217181451_provide_compiled_resource_bundles_for_plone_5/registry.xml

```diff
@@ -1,21 +1,19 @@
 <?xml version="1.0"?>
 <registry>
+
   <records prefix="plone.resources/throttledebounce"
-    interface="Products.CMFPlone.interfaces.IResourceRegistry">
-    <value key="js">++resource++jquery.ba-throttle-debounce.js</value>
-    <value key="deps">jquery</value>
-  </records>
+           interface="Products.CMFPlone.interfaces.IResourceRegistry"
+           remove="True" />
 
   <records prefix="plone.bundles/collective-js-throttledebounce-resources"
-    interface="Products.CMFPlone.interfaces.IBundleRegistry">
-    <value key="resources">
-      <element>throttledebounce</element>
-    </value>
-
+           interface="Products.CMFPlone.interfaces.IBundleRegistry">
     <value key="enabled">True</value>
-    <value key="depends">plone-legacy</value>
+    <value key="depends">plone</value>
+    <value key="stub_js_modules">jquery</value>
     <value key="compile">False</value>
     <value key="jscompilation">++plone++collective.js.throttledebounce/collective.js.throttledebounce-compiled.js</value>
```

Comment:
Same here...

Reply:
Oh - yes - that's true.  Should stop some TTW overwriting issues we don't want

Reply:
👍 thanks